src/arrayMethodSort.js

```diff
@@ -4,8 +4,31 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function compareDefault(a, b) {
+    if (String(a) > String(b)) {
+      return 1;
+    }
+
+    if (String(a) < String(b)) {
+      return -1;
+    }
+
+    return 0;
+  }
+
+  [].__proto__.sort2 = function (compareFunction = compareDefault) {
```

Comment:
The parameter 'compareFunction' is correctly provided with a default value. This is good practice as it ensures the method will work even if no custom compare function is passed.